test/tools/router_check/router.cc

```diff
@@ -0,0 +1,202 @@
+#include "router.h"
```

Comment:
#include "test/tools/router_check/router.h", we no longer allow relative includes in the style for Bazelish reasons.